projects/plugins/jetpack/functions.photon.php

```diff
@@ -3,339 +3,79 @@
  * Generic functions using the Photon service.
  *
  * Some are used outside of the Photon module being active, so intentionally not within the module.
+ * As photon has been moved to the image-cdn package, the functions are now also replaced by their counterparts in Image_CDN_Core in the package.
  *
  * @package automattic/jetpack
  */
 
-use Automattic\Jetpack\Status;
+use Automattic\Jetpack\Image_CDN\Image_CDN_Core;
 
 /**
  * Generates a Photon URL.
  *
  * @see https://developer.wordpress.com/docs/photon/
  *
+ * @deprecated 9.5 Use Automattic\Jetpack\Image_CDN\Image_CDN_Core::cdn_url instead.
```

Comment:
9.5? Aren't we up to Jetpack 12?

Reply:
Thank you for noticing. I thought I checked the current version, and not sure I ended up with 9.5. It appears the upcoming version is 12.1. I have update it.